db-async-common/src/main/java/com/github/jasync/sql/db/Configuration.kt

```diff
@@ -55,3 +55,10 @@ data class Configuration @JvmOverloads constructor(
     val eventLoopGroup: EventLoopGroup = NettyUtils.DefaultEventLoopGroup,
     val executionContext: Executor = ExecutorServiceUtils.CommonPool
 )
+
+fun Configuration.toDebugString(): String {
+    val toString = this.toString()
+    return toString.replace(passRegex, ", password=****, database=")
+}
+
+private val passRegex = ", password=.*, database=".toRegex()
```

Comment:
You probably want this to be non-greedy, otherwise it's possible that an awful applicationName breaks this.

Reply:
Thanks! I changed the implementation to copy data instead of non-greedy regexp.